### PR TITLE
Add subwatershed delineation `investspec` parameters

### PR DESCRIPTION
**Pull request by phargogh**
This is the UG counterpart to https://github.com/natcap/invest/pull/1291 for exposing subwatershed delineation to the user.

### CODE REVIEW

File: source/en/routedem.rst [hunk 1]
```diff
@@ -28,6 +28,10 @@ The freshwater models in InVEST are routed across a digital elevation model (DEM
 
 * Create streams based on flow accumulation.
 
+* Determine the Strahler Stream Order of streams.
```
Comment (dcdenu4) [inline source/en/routedem.rst:31]: I mentioned this over in the InVEST PR too but would it be worth elaborating on what Strahler Stream order is? I don't think it's mentioned at all in the UG. Or is it common knowledge enough for us not to have to have an explainer?

**Review by dcdenu4 — COMMENTED** [review]
Thanks @phargogh, just a quick comment / question!

**Comment by dcdenu4** [comment]
Oh, and should this go into a bugfix release branch? Since these changes won't go into effect until the next InVEST release?

File: source/en/routedem.rst (hunk 1 above)
Comment (phargogh) [reply source/en/routedem.rst:31]: Yeah, this is new in the InVEST context (not that we actually use Strahler orders in InVEST), but I think this is something that's pretty standard in hydrology.  It seems to me that if we need to explain Strahler stream orders, then we should probably also explain flow accumulation!  What do you think?

**Comment by phargogh** [comment]
@dcdenu4 thanks for the feedback!  I just updated the target branch.

File: source/en/routedem.rst (hunk 1 above)
Comment (dcdenu4) [reply source/en/routedem.rst:31]: Let's punt on it! Feels more like a science communication thing and it'd be great if we could defer to other NatCappers for that.

**Pull request merged**